modules/utxo-lib/src/payments/lazy.ts

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function prop(object: any, name: string, f: () => any): void {
```

Comment:
Didn't we expor this from bitcoinjs-lib instead?

Reply:
updated